src/rdposix.h

```diff
@@ -108,12 +126,22 @@
  */
 static RD_INLINE RD_UNUSED
 void rd_usleep (int usec, rd_atomic32_t *terminate) {
+#ifdef __370__
+    if(usec > 1000000)
+    {
```

Comment:
Since sleep() can be interrupted this needs to be wrapped in a similar loop as the one below.

Style guide promotes open-brace on same line.